src/safariservices.cs

```diff
@@ -15,10 +15,16 @@
 using ObjCRuntime;
 #if !MONOMAC
 using UIKit;
+using NSRunningApplication = System.Object;
+using NSImage = System.Object;
```

Comment:
UIKit.UIImage is the equivalent for AppKit.NSImage, so we can just use that instead:

```suggestion
using NSImage = UIKit.UIImage;
```

this ensures that if Apple ever exposes any API using `NSImage` from the API here on iOS, the bindings will be correct on iOS.

In other words: it's not necessary by any stretch, it's just an attempt to predict the future and avoid potential problems.